src/diskio/threaded.rs

```diff
@@ -268,6 +268,7 @@ impl<'a> Executor for Threaded<'a> {
                 prev_files as u64,
             ));
         }
+        #[allow(clippy::print_stderr)]
```

Comment:
Why should this be exempted from the logging policy?

Reply:
@djc Nice catch! It's not a deliberate exemption actually, it's just that I found it a bit hard to determine where to inject `process` in here, but I could be doing it wrong.

Update: It just occurred to me that, as `process`-injected `stderr` is part of the test result, it might make some sense to use the logger here with a lower priority level?

Reply:
Sounds okay to me.

Reply:
Addressed in #4063.